client/templates/topic/topicInfoItemEdit.js

```diff
@@ -18,19 +19,23 @@ import { emailAddressRegExpTest } from '/imports/helpers/email';
 import { $ } from 'meteor/jquery';
 import { handleError } from '/client/helpers/handleError';
 
-
 Session.setDefault('topicInfoItemEditTopicId', null);
 Session.setDefault('topicInfoItemEditInfoItemId', null);
 Session.setDefault('topicInfoItemType', 'infoItem');
 
 let _minutesID; // the ID of these minutes
 let _meetingSeries; // ATTENTION - this var. is not reactive! It is cached for performance reasons!
 
+collapseState = new ReactiveVar(false);
+
 Template.topicInfoItemEdit.onCreated(function () {
     _minutesID = this.data;
     console.log('Template topicEdit created with minutesID '+_minutesID);
     let aMin = new Minutes(_minutesID);
     _meetingSeries = new MeetingSeries(aMin.parentMeetingSeriesID());
+
+    const user = new User();
+    this.collapseState = new ReactiveVar(user.getSetting(userSettings.showAddDetail['topicInfoItemEdit'], true));
 });
```

Comment:
I don't understand this second level of the settings. If there is no forther subkey to "showAddDetail" - we do not need topicInfoItemEdit.